Resources/config/services.xml

```diff
@@ -4,84 +4,52 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
-    <parameters>
-        <parameter key="ets_global_log.tracing.token_collection.class">ETSGlobal\LogBundle\Tracing\TokenCollection</parameter>
-        <parameter key="ets_global_log.tracing.plugins.symfony.http_foundation.class">ETSGlobal\LogBundle\Tracing\Plugins\Symfony\HttpFoundation</parameter>
-        <parameter key="ets_global_log.tracing.plugins.guzzle.token_global_middleware.class">ETSGlobal\LogBundle\Tracing\Plugins\Guzzle\TokenGlobalMiddleware</parameter>
-        <parameter key="ets_global_log.event_subscriber.tracing.class">ETSGlobal\LogBundle\EventSubscriber\TracingEventSubscriber</parameter>
-        <parameter key="ets_global_log.monolog.processor.change_exceptions_log_level.class">ETSGlobal\LogBundle\Monolog\Processor\ChangeExceptionsLogLevelProcessor</parameter>
-        <parameter key="ets_global_log.monolog.processor.token_collection.class">ETSGlobal\LogBundle\Monolog\Processor\TokenCollectionProcessor</parameter>
-        <parameter key="ets_global_log.monolog.processor.extra_field.class">ETSGlobal\LogBundle\Monolog\Processor\ExtraFieldProcessor</parameter>
-        <parameter key="ets_global_log.monolog.formatter.token_collection.class">ETSGlobal\LogBundle\Monolog\Formatter\TokenCollectionFormatter</parameter>
-        <parameter key="ets_global_log.monolog.handler.content_data_modifier.add_class_line_context.class">ETSGlobal\LogBundle\Monolog\Handler\ContentDataModifier\AddClassLineContext</parameter>
-        <parameter key="ets_global_log.monolog.handler.content_data_modifier.add_jira_link.class">ETSGlobal\LogBundle\Monolog\Handler\ContentDataModifier\AddJiraLink</parameter>
-        <parameter key="ets_global_log.monolog.handler.content_data_modifier.add_kibana_token_filter_links.class">ETSGlobal\LogBundle\Monolog\Handler\ContentDataModifier\AddKibanaTokenFilterLinks</parameter>
-        <parameter key="ets_global_log.monolog.handler.exclusion_strategy.status_codes_http_exception.class">ETSGlobal\LogBundle\Monolog\Handler\ExclusionStrategy\StatusCodesHttpExceptionExclusionStrategy</parameter>
-        <parameter key="ets_global_log.monolog.handler.slack.class">ETSGlobal\LogBundle\Monolog\Handler\SlackHandler</parameter>
-        <parameter key="ets_global_log.monolog.handler.ignore_deprecation.class">ETSGlobal\LogBundle\Monolog\Handler\IgnoreDeprecationHandler</parameter>
-    </parameters>
-
```

Comment:
are you sure we never use these parameters to override one of these classes in our microservices?

Reply:
No (just check emo) but we want to let ms to overload this parameters ?

Reply:
if no need to override, ok